docs-2.0/graph-computing/0.deploy-controller-analytics.md

```diff
@@ -124,7 +124,17 @@ There are some differences between installation packages and commands in differe
     VarDataListMaxSize: 100    # If HDFS columns are read, the number is limited to 100 at a time.
   ```
 
-5. Configure the algorithm file path (`exec_file`) in the file `tasks.yaml`, the file path is `dag-ctrl/etc/tasks.yaml`. If there are multiple machines, ensure that the paths are the same.
+5. Configure the algorithm file path (`exec_file`) only in the file `tasks.yaml`, the file path is `dag-ctrl/etc/tasks.yaml`. Currently, all `exec_file` parameters are set to the path of the `run_algo.sh` file.
+
+  !!! note
+
+        - The algorithm files are provided by NebulaGraph Analytics. Please find directory `scripts` under the installation path of NebulaGraph Analytics above. All algorithm files are in this directory.  
```

Comment:
directory `scripts`  --the directory of `scripts` or the `scripts` directory

ps: do we need to note the path of the `scripts` directory?